core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -101,13 +101,17 @@
                        angular-html-bind="activeCard.interactionHtml">
                   </div>
                 </div>
-                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)"
-                     style="opacity: 0.8;">
-                  <md-button class="instructions-button"
-                          ng-click="showSupplementalCardIfScreenIsNarrow()">
+                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)" style="opacity: 0.8;">
+                  <div ng-if="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" style="text-transform: uppercase; padding: 6px 12px;"> 
```

Comment:
I don't think you need "text-transform: uppercase;" here -- it's a bit "shouty". Can you try it without?